src/index.js

```diff
@@ -121,6 +118,7 @@ function EEAClient(web3, chainId) {
    * - options.nonce
    * - options.to
    * - options.data
```

Comment:
add the `gasPrice` and `gasLimit` to the JSDoc and their defaults value

Reply:
Done, please review it.